islandora_solr.install

```diff
@@ -234,6 +234,54 @@ function islandora_solr_update_7001() {
   db_create_table('islandora_solr_collection_sort_strings', $schema['islandora_solr_collection_sort_strings']);
 }
 
+/**
+ * Fix the table islandora_solr_collection_sort_strings, if possible.
+ */
+function islandora_solr_update_7002() {
+  module_load_include('inc', 'islandora', 'includes/utilities');
+  $schema = islandora_solr_schema();
+  $results = db_select('islandora_solr_collection_sort_strings', 'c')
+    ->fields('c', array('collection_pid', 'sort_string'))
+    ->execute();
+
+  $is_empty = function($o) {
+    return (empty($o) || trim($o) == FALSE);
```

Comment:
I have no problem with that. I was looking at the empty manual page as I wrote this and saw a note on there suggesting this usage with a PHP version < 5.5. I'll add a commit.